locations/spiders/bureau_vallee_fr.py

```
@@ -0,0 +1,17 @@
+
+from locations.categories import Categories
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class BureauValleeFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "bureau_vallee_fr"
+    item_attributes = {
+        "brand": "Bureau Vallée",
+        "brand_wikidata": "Q18385014",
+        "extras": Categories.SHOP_STATIONERY.value,
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)